content/docs/09-registries-and-packs/4.5-create-pack.md

```diff
@@ -0,0 +1,718 @@
+
+<br />
+
+### Section: Profile Layers
+In the **Profile Layers** section, add the following core infrastructure layers:
+
+|**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+|---|---|---|---|
+|OS|Public Repo|Ubuntu|`LTS__20.4.x`|
+|Kubernetes|Public Repo|Kubernetes|`1.24.x`|
+|Network|Public Repo|Calico|`3.25.x`|
+|Storage|Public Repo|Amazon EBS CSI|`1.16.x`|
+
+Once you confirm adding the core infrastructure layers, click the **Add New Pack** button to add the following layers (including the custom add-on pack you defined and pushed to the registry). 
+
+|**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+|---|---|---|---|
+|Authentication | Public Repo| Spectro Proxy | `1.3.x`|
+| App Services | **private-pack-registry** | **Hello Universe** | `1.0.x` |
+
+Note that the values, **private-pack-registry** and the **Hello Universe**, may vary if you have chosen a different name for your registry and pack, respectively. 
+
+
+<br />
+
+![Visual of the Profile Layers success.](/tutorials/create-pack/registries-and-packs_create-pack_profile-layer.png)
+
+<br /> <br />
+
+
+### Section: Review
+Lastly, review and finish creating the profile configuration. You would see the newly created profile if there were no errors or compatibility issues between the individual layers. 
+
+<br /> <br />
+
+## Create a Cluster
+View the details of the profile you just created, and click the **Deploy** button on the top right to create a new cluster. The cluster creation wizard will present you with the a few sections. 
+<br />
+
+### Section: Basic Information
+Use the following values in the first section, **Basic Information**. 
+
+|**Field**|**Value**|
+|---|---|
+|Cluster name| *pack-tutorial-cluster* |
+|Description| *Cluster as part of the pack tutorial.*|
+|Tags|`spectro-cloud-education, app:hello-universe, terraform_managed:true`|
+|Cloud Account|**spectro-cloud**|
+
+Note that the value **spectro-cloud** is the AWS cloud account name already configured in the Palette project settings. 
+
+<br /> <br />
+
+### Section: Parameters
+Next, the **Parameters** section allows you to change the profile configurations. For example, clicking on the **Hello Universe 1.0.x** layer will allow you to configure the `registry`, `repository`, and `tag` parameters defined in the **values.yaml** file. Let's keep those parameter values as default and move to the next section. 
+
+<br /> <br />
+
+### Section: Cluster config
+In the **Cluster config** section, uncheck the **Static Placement** field, select the **us-east-2** region, and select the SSH key pair name you have in your AWS console. If checked, the **Static Placement** will deploy the cluster in an existing VPC, subnets, roles, and other resources. But, for this tutorial, you will choose the dynamic placement, meaning the Palette dev engine will create a new VPC and all other resources needed for the cluster. 
```

Comment:
Tell them to choose a region where they have an SSH key. We shouldn't assume us-east-2.